web/layouts/property/property-tabs.html

```
@@ -0,0 +1,85 @@
+{{define "property-tabs.html"}}
+{{ $root := .Root }}
+{{ $active := .ActiveTab }}
+<div>
+    <div class="sm:hidden">
+        <label for="tabs" class="sr-only">Select a tab</label>
+        <select id="tabs" name="tabs" class="pc-internal-form-select"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)